indexer/src/main/java/org/pdxfinder/dataloaders/updog/domainobjectcreation/DomainObjectCreator.java

```diff
@@ -425,8 +434,7 @@ private void createCytogeneticsData(Map<String, Table> pdxDataTables){
 
     private void createMolecularData(Table table, String molcharType){
 
-
-        MarkerAssociation markerAssociation = null;
+       MarkerAssociation markerAssociation;
         for (Row row : table) {
 
             if(row.getRowNumber() != 1){
```

Comment:
This if statements needs to be removed, it will not load the first record otherwise